### ISSUE TITLE
How to use SyncBatchNorm implemented inside mmcv

### ISSUE THREAD

**Issue by xpngzhng** [open-mmlab/mmcv#418]
in 
https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/bricks/norm.py
the sync batch morm comes from mmcv.utils.parrots_wrapper (https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/parrots_wrapper.py)
inside which the standard pytorch sync batch norm is imported

so, how to use sync batch norm implemented inside mmcv
If I have CUDA, is it still OK to install mmcv-lite for mmdetection, mmclassification, and other mm libraries?

**Comment by xvjiarui** [comment]
Hi @xpngzhng 
To use SyncBn in MMCV, you just need to specify `norm_cfg=dict(type='MMSyncBN')`.
Some repos need CUDA ops, some not. For example, MMDetection requires `mmcv-full`, MMClassification not.
